src/theme/formStyles.js

```diff
@@ -119,11 +119,11 @@ const inputStyle = {
 	padding: '5px 10px',
 
 	'&:focus': {
-		outline: 'none',
+		border: 'none',
 	},
 
 	'&:focus-visible': {
-		outline: `solid 1px ${primary.default}`,
+		border: `solid 2px ${primary.default}`,
 
```

Comment:
Add `outline: none` to make the default browser outline disappear (there is a weird blue outline visible on mac os firefox)

Reply:
Added `outline: none` which should affect the error state as well